docs/jwst/mrs_imatch/description.rst

```diff
@@ -8,6 +8,10 @@ Description
 
 Overview
 --------
+
+This step has been deprecated in JWST calibration pipeline version 11.1.
```

Comment:
Sorry, I missed this earlier.  The build will be 11.1, the pipeline version is not yet known.  Maybe we should revise this to just say that the step is deprecated and will be skipped by default?  They can refer to the change log if they need to know when it became deprecated.

Reply:
done